### PR TITLE
Interpolate translation array

### PR DESCRIPTION
**Pull request by c100k**
If the translation is an `Array`, the dynamic values were not interpolated.

### CODE REVIEW

**Comment by PikachuEXE** [comment]
Can you add related test to prove your change is working?

**Comment by c100k** [comment]
Done ! Let me know if something more is needed.

File: app/assets/javascripts/i18n.js [hunk 1]
```diff
@@ -598,6 +598,8 @@
 
     if (typeof(translation) === "string") {
       translation = this.interpolate(translation, options);
+    } else if (isArray(translation)) {
+      translation = translation.map((t) => this.interpolate(t, options));
```
Comment (PikachuEXE) [inline app/assets/javascripts/i18n.js]: 1. This `() => something` syntax is not ES5 syntax, please update it
2. Please add shim for `Array#map` to `app/assets/javascripts/i18n/shims.js`

**Comment by c100k** [comment]
Done.

**Comment by PikachuEXE** [comment]
Thanks!

**Comment by PikachuEXE** [comment]
Released as gem and npm package `3.0.8`

**Pull request merged**